src/lib/storage/fixed_size_attribute_vector.hpp

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <vector>
+
+#include "base_attribute_vector.hpp"
+#include "types.hpp"
+
+namespace opossum {
+
+template <typename T>
+class FixedSizeAttributeVector : public BaseAttributeVector {
+ public:
+  FixedSizeAttributeVector() = default;
+  ~FixedSizeAttributeVector() override = default;
+
+  // we need to explicitly set the move constructor to default when
+  // we overwrite the copy constructor
+  FixedSizeAttributeVector(FixedSizeAttributeVector&&) = default;
+  FixedSizeAttributeVector& operator=(FixedSizeAttributeVector&&) = default;
+
+  // returns the value id at a given position
+  ValueID get(const size_t i) const override { return static_cast<ValueID>(_vector[i]); };
```

Comment:
It should be checked if the position exists before accessing it with the [] operator (like you do chunk.cpp, line 28f.)